msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2018-02-19 03:59+UTC\n"
"PO-Revision-Date: 2018-02-25 21:33-0500\n"
"Last-Translator: Kowlin <boxedpp@gmail.com>\n"
"Language-Team: Portuguese\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Language: pt-PT\n"
"X-Crowdin-File: /core/locales/messages.pot\n"
"Language: pt_PT\n"

#: ../cog_manager.py:309
msgid "Install Path: {}\n\n"
msgstr ""

#: ../cog_manager.py:325
msgid "That path does not exist or does not point to a valid directory."
msgstr ""

#: ../cog_manager.py:334
msgid "Path successfully added."
msgstr ""

#: ../cog_manager.py:347
msgid "That is an invalid path number."
msgstr ""

#: ../cog_manager.py:351
msgid "Path successfully removed."
msgstr ""

#: ../cog_manager.py:367
msgid "Invalid 'from' index."
msgstr ""

#: ../cog_manager.py:373
msgid "Invalid 'to' index."
msgstr ""

#: ../cog_manager.py:377
msgid "Paths reordered."
msgstr ""

#: ../cog_manager.py:395
msgid "That path does not exist."
msgstr ""

#: ../cog_manager.py:399
msgid "The bot will install new cogs to the `{}` directory."
msgstr ""

#: ../core_commands.py:225 ../core_commands.py:266
msgid "No module by that name was found in any cog path."
msgstr ""

#: ../core_commands.py:240
msgid "Failed to load package. Check your console or logs for details."
msgstr ""

#: ../core_commands.py:244 ../core_commands.py:253 ../core_commands.py:285
#: ../core_commands.py:374 ../core_commands.py:462 ../core_commands.py:476
msgid "Done."
msgstr ""

#: ../core_commands.py:255
msgid "That extension is not loaded."
msgstr ""

#: ../core_commands.py:282
msgid "Failed to reload package. Check your console or logs for details."
msgstr ""

#: ../core_commands.py:295
msgid "Shutting down... "
msgstr ""

#: ../core_commands.py:310
msgid "Restarting..."
msgstr ""

#: ../core_commands.py:347
msgid "The admin role for this guild has been set."
msgstr ""

#: ../core_commands.py:355
msgid "The mod role for this guild has been set."
msgstr ""

#: ../core_commands.py:368
msgid "Failed. Remember that you can edit my avatar up to two times a hour. The URL must be a direct link to a JPG / PNG."
msgstr ""

#: ../core_commands.py:372
msgid "JPG / PNG format only."
msgstr ""

#: ../core_commands.py:387
msgid "Game set."
msgstr ""

#: ../core_commands.py:399
msgid "Listening set."
msgstr ""

#: ../core_commands.py:411
msgid "Watching set."
msgstr ""

#: ../core_commands.py:440
msgid "Status changed to %s."
msgstr ""

#: ../core_commands.py:471
msgid "Failed to change name. Remember that you can only do it up to 2 times an hour. Use nicknames if you need frequent changes. `{}set nickname`"
msgstr ""

#: ../core_commands.py:486
msgid "I lack the permissions to change my own nickname."
msgstr ""

#: ../core_commands.py:500 ../core_commands.py:513
msgid "Prefix set."
msgstr ""

#: ../core_commands.py:509
msgid "Guild prefixes have been reset."
msgstr ""

#: ../core_commands.py:532
msgid "\n"
"Verification token:"
msgstr ""

#: ../core_commands.py:535
msgid "Remember:\n"
msgstr ""

#: ../core_commands.py:538
msgid "I have printed a one-time token in the console. Copy and paste it here to confirm you are the owner."
msgstr ""

#: ../core_commands.py:546
msgid "The set owner request has timed out."
msgstr ""

#: ../core_commands.py:552
msgid "You have been set as owner."
msgstr ""

#: ../core_commands.py:554
msgid "Invalid token."
msgstr ""

#: ../core_commands.py:566
msgid "Locale has been set."
msgstr ""

#: ../core_commands.py:580
msgid "Done. Sentry logging is now enabled."
msgstr ""

#: ../core_commands.py:583
msgid "Done. Sentry logging is now disabled."
msgstr ""

#: ../core_commands.py:593
msgid "User ID: %s"
msgstr ""

#: ../core_commands.py:596
msgid "through DM"
msgstr ""

#: ../core_commands.py:598
msgid "from {}"
msgstr ""

#: ../core_commands.py:599
msgid " | Server ID: %s"
msgstr ""

#: ../core_commands.py:608
msgid "Use `{}dm {} <text>` to reply to this user"
msgstr ""

#: ../core_commands.py:616
msgid "Sent by {} {}"
msgstr ""

#: ../core_commands.py:628
msgid "I cannot send your message, I'm unable to find my owner... *sigh*"
msgstr ""

#: ../core_commands.py:631
msgid "I'm unable to deliver your message. Sorry."
msgstr ""

#: ../core_commands.py:633
msgid "Your message has been sent."
msgstr ""

#: ../core_commands.py:647
msgid "Invalid ID or user not found. You can only send messages to people I share a server with."
msgstr ""

#: ../core_commands.py:653
msgid "Owner of %s"
msgstr ""

#: ../core_commands.py:657
msgid "You can reply to this message with %scontact"
msgstr ""

#: ../core_commands.py:667
msgid "Sorry, I couldn't deliver your message to %s"
msgstr ""

#: ../core_commands.py:670
msgid "Message delivered to %s"
msgstr ""

#: ../core_commands.py:690
msgid "User added to whitelist."
msgstr ""

#: ../core_commands.py:699
msgid "Whitelisted Users:"
msgstr ""

#: ../core_commands.py:719
msgid "User has been removed from whitelist."
msgstr ""

#: ../core_commands.py:721
msgid "User was not in the whitelist."
msgstr ""

#: ../core_commands.py:729
msgid "Whitelist has been cleared."
msgstr ""

#: ../core_commands.py:746
msgid "You cannot blacklist an owner!"
msgstr ""

#: ../core_commands.py:753
msgid "User added to blacklist."
msgstr ""

#: ../core_commands.py:762
msgid "blacklisted Users:"
msgstr ""

#: ../core_commands.py:782
msgid "User has been removed from blacklist."
msgstr ""

#: ../core_commands.py:784
msgid "User was not in the blacklist."
msgstr ""

#: ../core_commands.py:792
msgid "blacklist has been cleared."
msgstr ""

#: ../dev_commands.py:213
msgid "Already running a REPL session in this channel. Exit it with `quit`."
msgstr ""

#: ../dev_commands.py:218
msgid "Enter code to execute or evaluate. `exit()` or `quit` to exit."
msgstr ""

#: ../dev_commands.py:281
msgid "Unexpected error: `{}`"
msgstr ""

